stable/consul/templates/consul.yaml

```diff
@@ -157,17 +182,11 @@ spec:
       - name: gossip-key
         secret:
           secretName: gossip-key
-      readinessProbe:
-        httpGet:
-          path: /v1/health/service/consul
-          port: {{.Values.HttpPort}}
-        initialDelaySeconds: 10
-        timeoutSeconds: 1
   volumeClaimTemplates:
   - metadata:
       name: datadir
       annotations:
-        volume.alpha.kubernetes.io/storage-class: anything
+        volume.beta.kubernetes.io/storage-class: "{{.Values.StorageClass}}"
```

Comment:
The change is more about the default storage class than alpha vs beta features. With the alpha you don't need a default cluster level storage class while with the beta you do. Until we have more clusters that have default storage classes I think we should use the alpha+beta pattern.